butter-cms.json

```diff
@@ -30,54 +30,54 @@
     "Search Features [Geospatial Search]": "No",
     "Search Features [Search within uploads (pdf,doc)]": "No",
     "Image Manipulation": "Yes",
-    "Image Focalpoint support": "No",
-    "Image Facedetection": "No",
+    "Image Focalpoint support": "Yes",
+    "Image Facedetection": "Yes",
     "Optimized Image Encoding": "Yes",
-    "Asset fingerprinting": "?",
+    "Asset fingerprinting": "No",
     "CDN Support": "Yes",
     "Antivirus Scanning": "No",
     "Custom Binary Handler": "No",
     "System Requirements": "NA",
     "Supported Databases": "NA",
     "Clustering": "NA",
     "Docker Support": "NA",
-    "Backup Feature": "No",
-    "Import/Export": "No",
-    "CLI": "No",
-    "SDK [Java]": "No",
+    "Backup Feature": "Yes",
```

Comment:
Butter does automatic backups. https://buttercms.com/security/

Reply:
Unless you can download the backup I don't see this as a valid feature.

Reply:
@Jotschi you can download backups if the ButterCMS account is on an Enterprise Plan (they setup auto dumps to account holders managed S3 buckets).